mir-layout/src/main/resources/META-INF/resources/mir-layout/scss/common/mir_browse.scss

```diff
@@ -143,4 +143,10 @@ ul ul span.cbNum {
         word-break: break-all;
         padding-right: 20px;
     }
+
+    .md5 {
```

Comment:
I suggest to rename that class to `file_md5` to it follows the same rules as the other file related classed

Reply:
fixed